src/main/java/success/planfit/domain/user/PlanfitUser.java

```diff
@@ -0,0 +1,34 @@
+package success.planfit.domain.user;
+
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class PlanfitUser {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(nullable = false)
+    private int user_id;
```

Comment:
```suggestion
    @JoinColumn(nullable = false)
    @ManyToOne(fetch = FetchType.LAZY)
    private User user;
```

JPA에서 제공해주는 애노태이션인 `@ManyToOne`을 사용해 `User` 객체와 직접 연결해 주실 수 있나요?
객체 그래프를 통해 탐색할 수 있도록 하는 것이 객체지향에 적합할 것 같습니다.
다른 엔티티들도 위 예시처럼 수정해 주시면 감사하겠습니다!

또한 위 예시처럼 필드명을 `user`로만 작성해 주셔도, JPA가 자동으로 DB의 `user_id` 컬럼과 매핑시켜 준다는 것도 참고하시면 좋을 것 같습니다!